lxd/instance_backup.go

```diff
@@ -528,9 +528,11 @@ func instanceBackupPost(d *Daemon, r *http.Request) response.Response {
 		return nil
 	}
 
-	resources := map[string][]string{}
-	resources["instances"] = []string{name}
-	resources["containers"] = resources["instances"]
+	resources := map[string][]api.URL{}
+	resources["instances"] = []api.URL{*api.NewURL().Path(version.APIVersion, "instances", name, "backups", newName)}
```

Comment:
`newName` will be `<name>/<backupName>`. Shouldn't we just use `req.Name` here?

Reply:
Agreed.

Reply:
Also the URL is incorrect because the URL represents an instance backup, but its being stored in a category key of "instances". We should retain the "instances" resource with correct URL and add the backup URL as a separate category.

Reply:
Ah, checked above and we are using "backups" as resources key for instance backups.